controller_manager/controller_manager/spawner.py

```diff
@@ -141,6 +141,13 @@ def main(args=None):
         action="store_true",
         required=False,
     )
+    parser.add_argument(
```

Comment:
this means that you change the default behavior, right?

Reply:
Yes, for the rolling that's the idea. In ros_control that's the default behavior. I'm also fine to change it back to true.

Reply:
Sure thing 🫡